app/jsx/pages/UnitPage.jsx

```diff
@@ -16,15 +18,20 @@ class UnitPage extends PageBase
 
   renderData(data) { return(
     <div>
-      <HeaderComp level="unit"
-                  isJournal = {data.isJournal}
-                  campusID={data.campusID}
-                  campusName={data.campusName}
-                  campuses={data.campuses}
-                  unit_id={data.id} />
-      <NavComp level="unit"
-               campusID={data.campusID}
-               campuses={data.campuses} />
+      <Header2Comp type={data.type} unitID={data.id} /> 
+      { data.type == "journal" &&
+          <Subheader2Comp unitID={data.id}
+                          unitName={data.name}
+                          campusID={data.campusID}
+                          campusName={data.campusName}
+                          campuses={data.campuses} /> }
```

Comment:
This paradigm of using `condition && <Component>` is quite elegant and compact. Nice find!